tests/testPhotoCal.py

```diff
@@ -125,6 +124,31 @@ def _runTask(self):
         self.zp = pCal.calib.getMagnitude(1.)
         self.fitdiff = pCal.arrays.srcMag + self.zp - pCal.arrays.refMag
 
+    def testFlags(self):
+        """test that all the calib_photometry flags are set to reasonable values"""
+        schema = self.srcCat.schema
+        self.config.doWriteOutput = True
+        task = PhotoCalTask(self.refObjLoader, config=self.config, schema=schema)
+        mapper = afwTable.SchemaMapper(self.srcCat.schema, schema)
+        cat = afwTable.SourceCatalog(schema)
+        for name in self.srcCat.schema.getNames():
+            mapper.addMapping(self.srcCat.schema.find(name).key)
+        cat.extend(self.srcCat, mapper=mapper)
+
+        #   test that by default, no stars are reserved and used < candidates
+        task.run(exposure=self.exposure, sourceCat=cat)
+        candidates = cat.get("calib_photometryCandidate").sum()
+        assert(cat.get("calib_photometryReserved").sum() == 0)
+        assert(cat.get("calib_photometryUsed").sum() <= candidates)
+
+        #   set the reserve fraction, and see if the right proportion are reserved.
+        self.config.doWriteOutput = True
```

Comment:
This usage here strongly suggests that `doWriteOutput` is the wrong name for that option. If you don't want to change it on this ticket, please file another ticket to change it in the future.